packages/cicero-core/package.json

```diff
@@ -90,8 +91,7 @@
     "moment-mini": "2.22.1",
     "node-cache": "4.2.0",
     "request": "2.88.0",
-    "request-promise-native": "1.0.7",
-    "semver": "6.3.0",
+    "semver": "6.2.0",
```

Comment:
Why the version downgrade?

Reply:
Also: should `axios` be added? Which version should we use (would be worth checking which version is used in Concerto so they are in sync).

Reply:
@jeromesimeon Have a look upon [this comment](https://github.com/accordproject/cicero/pull/623#issuecomment-809313894).Thats why @mttrbrts shifted to axios since we were unable to test it via react-promise-native.

Reply:
I am clueless about this downgraded version of `semver`.

Reply:
- I'm only saying that if we have axios as a dependency, it should be listed in the dependencies (in the `package.json` with a specific version)
- Then let's _not_ downgrade `semver`

Reply:
@mttrbrts Have a look upon [this](https://github.com/accordproject/cicero/commit/4555788b4fce67e495d141455c2f04de3617d31f#r49121242)

Reply:
@jeromesimeon The axios version is [0.21.1](https://github.com/accordproject/concerto/blob/master/packages/concerto-core/package.json#L66) same as that of [cicero-core](https://github.com/accordproject/cicero/blob/master/packages/cicero-core/package.json#L85)

Reply:
`axios` was already a dependency. That's why I removed `request-promise-native`

> Migrates from request-promise-native to axios to ensure consistency of HTTP libraries and to reduce dependencies

Reply:
> `axios` was already a dependency. That's why I removed `request-promise-native`
> 
> > Migrates from request-promise-native to axios to ensure consistency of HTTP libraries and to reduce dependencies

oh yes I see it. I got confused with the diff I think. that makes sense 👍